### PR TITLE
Misc Updates

### PR DESCRIPTION
**Pull request by ShellyXueHan**
1. update url

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -54,7 +54,8 @@ export const SELF_SERVER_APP = {
     INVITATION_CODE: 'rc',
   },
   REGGIE: {
-    README: 'https://github.com/bcgov/reggie-web/blob/master/README-user-instructions.md',
+    README:
+      'https://developer.gov.bc.ca/Community-Enablers-and-Events/BC-Gov-Development-Community-Events',
```
Comment (patricksimonian) [inline src/constants.js]: looks like this path is incorrect? 
It should be pointing towards the title of the source for the user instructions?
`Steps-to-join-Pathfinder-Rocket.Chat`

**Review by patricksimonian — CHANGES_REQUESTED** [review]

File: src/constants.js (hunk 1 above)
Comment (patricksimonian) [reply src/constants.js]: lets double check prod to get the right link of course

Comment (patricksimonian) [reply src/constants.js]: prod of developer.gov.bc.ca

**Comment by ShellyXueHan** [comment]
@patricksimonian ready for review

**Review by patricksimonian — APPROVED** [review]

**Pull request merged**